_security/configuration/best-practices.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: default
+title: Best Practices
+parent: Best Practices
+nav_order: 11
+---
+
+# Best Practices for OpenSearch Security plugin
```

Comment:
Are these best practices specific to the plugin or for securing your OpenSearch cluster as a whole?

```suggestion
# Best Practices for OpenSearch Security
```